src/math/decibels.c

```diff
@@ -9,42 +9,8 @@
 #include <sof/audio/format.h>
 #include <stdint.h>
 
-#define ONE_Q23         Q_CONVERT_FLOAT(1.0, 23)	  /* Use Q9.23 */
 #define LOG10_DIV20_Q27 Q_CONVERT_FLOAT(0.1151292546, 27) /* Use Q5.27 */
 
-/* Exponent function for small values of x. This function calculates
```

Comment:
Don't remove this. Both exponent functions should be possible to enable via kconfig. To be able to select both until we can safely remove the slower they should have different function name. DRC should use this new fast function.

Can your exponent function have different name, like have _fast_ in function name or some algorithm specific keyword to differentiate?  Or rename exp_fixed() and all usages of it (except DRC)  to e.g. exp_taylor_fixed(), since it's using Taylor -series.